plugins/git-release-manager/.eslintrc.js

```diff
@@ -1 +1,5 @@
-module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+    rules: {
+      '@backstage/no-top-level-material-ui-4-imports': 'error',
+    },
+  });
```

Comment:
Remember to enable the option in your editor that automatically adds newlines at the end of files 

Also `patch` above